lib/agris/api/messages/query_base.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+module Agris
+  module Api
+    module Messages
+      class QueryBase < MessageBase
+        protected
+
+        def input_base_hash
+          {
+            :@requester => 'Otis (Agris Client)',
```

Comment:
Yep, good catch. I agree we should make the agent configurable as with Ottokit. Same story but a separate PR? I've changed it to "Agris Client" for now.